lib/edge_impulse/ei_wrapper.cpp

```diff
@@ -319,9 +319,21 @@ static void edge_impulse_thread_fn(void)
 		features_signal.get_data = &raw_feature_get_data;
 		features_signal.total_length = INPUT_WINDOW_SIZE;
 
+#if IS_ENABLED(CONFIG_EI_WRAPPER_PROFILING)
```

Comment:
I actually thought about something like that

```
   int32_t start_time;

   if (IS_ENABLED(CONFIG_EI_WRAPPER_PROFILING)) {
      start_time = k_uptime_get_32();
   }
   do_stuff();
   if (IS_ENABLED(CONFIG_EI_WRAPPER_PROFILING)) {
      int32_t delta = k_uptime_get_32() - start_time;

      logme(delta);
   }
```

This way code is always checked during compilation - even if feature is disabled ;)